.github/workflows/image_build.yml

```diff
@@ -23,8 +23,8 @@ jobs:
         run: |
           TAG=$(git describe --tags --abbrev=0 --match 'v*' 2> /dev/null) || true
           if [ -z "$TAG" ]; then
-            echo "No tag found, use v0.1.0 as default"
```

Comment:
Since we've already released v0.1.0, the value of the TAG will be `v0.1.0`, and if we want to release the `v0.2.0` version, we'll just need the administrator to push tag `v0.2.0` to upstream, which shouldn't need to be changed here, IMO.